client/homescreen/activity-panel/orders/index.js

```diff
@@ -344,6 +340,7 @@ export default withSelect( ( select, props ) => {
 		per_page: 5,
 		extended_info: true,
 		order_includes: map( actionableOrders, 'id' ),
+		status_is: orderStatuses,
```

Comment:
Should this be `statuses:`?

Reply:
Hi @samueljseay, thank you for your review. Inside the `src/API` the variable name is: [status_is](https://github.com/woocommerce/woocommerce-admin/blob/fix/5567/src/API/Reports/Orders/Controller.php#L58). If I use `statuses` it won't work as expected.

Reply:
Thats all good was just checking if it was a typo or not :+1: